### PR TITLE
Release v1.1.24.0

### PR DESCRIPTION
**Pull request by iulianpascalau**
Release v1.1.24.0

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -314,6 +369,12 @@
     Type = "SizeLRU"
     SizeInBytes = 314572800 #300MB
 
+[SmartContractDataPool]
+    Name = "SmartContractDataPool"
+    Capacity = 900000
+    Type = "SizeLRU"
+    SizeInBytes = 419430400 #400MB
```
Comment (sasurobert) [inline config.toml]: change to 200mb.

File: config.toml [hunk 2]
```diff
@@ -314,6 +369,12 @@
     Type = "SizeLRU"
     SizeInBytes = 314572800 #300MB
 
+[SmartContractDataPool]
+    Name = "SmartContractDataPool"
+    Capacity = 900000
+    Type = "SizeLRU"
+    SizeInBytes = 209715200 #400MB
```
Comment (sasurobert) [inline config.toml:376]: change comment

**Review by sasurobert — CHANGES_REQUESTED** [review]

**Review by sasurobert — APPROVED** [review]

File: config.toml (hunk 1 above)
Comment (iulianpascalau) [reply config.toml]: done

File: config.toml (hunk 2 above)
Comment (iulianpascalau) [reply config.toml]: done

**Review by AdoAdoAdo — APPROVED** [review]

File: systemSmartContractsConfig.toml [hunk 3]
```diff
@@ -25,3 +26,14 @@
...
     EnabledEpoch = 1000000
+
+[DelegationManagerSystemSCConfig]
+    BaseIssuingCost = "0" #0 eGLD
+    MinCreationDeposit = "1250000000000000000000" #1.25K eGLD
+    EnabledEpoch = 1000000 #enable epoch should not be 0
+
+[DelegationSystemSCConfig]
+    MinStakeAmount = "10000000000000000000" #10 eGLD
+    EnabledEpoch   = 1000000 #enable epoch should not be 0
```
Comment (bogdan-rosianu) [inline systemSmartContractsConfig.toml:37]: i think you can remove this comment. also from above

**Review by bogdan-rosianu — APPROVED** [review]

File: systemSmartContractsConfig.toml (hunk 3 above)
Comment (iulianpascalau) [reply systemSmartContractsConfig.toml:37]: will remain like this for now

**Review by raduchis — APPROVED** [review]

File: config.toml [hunk 4]
```diff
@@ -555,15 +618,17 @@
...
-        WarmInstanceEnabled = false
+        OutOfProcessEnabled = false
         [VirtualMachine.Execution.OutOfProcessConfig]
             LogsMarshalizer = "json"
             MessagesMarshalizer = "json"
             MaxLoopTime = 1000
     [VirtualMachine.Querying]
-        OutOfProcessEnabled = false
-        WarmInstanceEnabled = true
+        NumConcurrentVMs = 1
```
Comment (SebastianMarian) [inline config.toml:627]: Only one?, not need for OutOfProcessEnabled = true for querying ?

**Pull request merged**